src/components/AddressSearch/index.tsx

```diff
@@ -343,14 +376,21 @@ function AddressSearch(
                         listEmptyComponent={listEmptyComponent}
                         listLoaderComponent={listLoader}
                         renderHeaderComponent={renderHeaderComponent}
-                        renderRow={(data) => {
+                        renderRow={(data, index) => {
                             const title = data.isPredefinedPlace ? data.name : data.structured_formatting.main_text;
                             const subtitle = data.isPredefinedPlace ? data.description : data.structured_formatting.secondary_text;
+                            if (focusedIndex === index) {
+                                setFocusedIndex(index);
```

Comment:
I don't think this should be necessary. `setFocusedIndex` is mostly returned from `useArrowKeyFocusManager` if you need to reset it to `-1`